watertap/tests/test_pure_water_pH.py

```diff
@@ -598,67 +582,35 @@ def test_initialize_inherent(self, inherent_reactions_config):
+        data_after = _collect_data_to_check(model)
+        return model, data_before, data_after
 
     @pytest.mark.component
-    def test_initialize_equilibrium(self, equilibrium_reactions_config):
-        model = equilibrium_reactions_config
-        state_args = {
-            "mole_frac_comp": {
-                "H2O": 1,
-                "H_+": 10 ** -7 / 55.6,
-                "OH_-": 10 ** -7 / 55.6,
-            },
-            "pressure": 101325,
-            "temperature": 298,
-            "flow_mol": 10,
-        }
-        orig_fixed_vars = fixed_variables_set(model)
-        orig_act_consts = activated_constraints_set(model)
-
-        model.fs.unit.control_volume.properties_out[0.0].log_mole_frac_phase_comp_true.setlb(-50)
-        model.fs.unit.control_volume.properties_out[0.0].log_mole_frac_phase_comp_true.setub(0.001)
-        model.fs.unit.control_volume.properties_out[0.0].mole_frac_phase_comp.setub(1.001)
-
-        solver.options["bound_push"] = 1e-5
-        solver.options["mu_init"] = 1e-3
-        model.fs.unit.initialize(state_args=state_args, optarg=solver.options)
-
-        fin_fixed_vars = fixed_variables_set(model)
-        fin_act_consts = activated_constraints_set(model)
+    def test_initialize(self, model_initialize):
+        model, data_before, data_after = model_initialize
 
         assert degrees_of_freedom(model) == 0
 
-        assert len(fin_act_consts) == len(orig_act_consts)
-        assert len(fin_fixed_vars) == len(orig_fixed_vars)
+        for key, value_before in data_before.items():
+            value_after = data_after[key]
+            assert len(value_before) == len(value_after)
 
-    @pytest.mark.component
-    def test_solve_inherent(self, inherent_reactions_config):
-        model = inherent_reactions_config
-        solver.options["max_iter"] = 20
+    @pytest.fixture
+    def model_solve(self, model, solver):
         results = solver.solve(model, tee=True)
-        assert results.solver.termination_condition == TerminationCondition.optimal
-        assert results.solver.status == SolverStatus.ok
+        return model, results
 
     @pytest.mark.component
-    def test_solve_equilibrium(self, equilibrium_reactions_config):
-        model = equilibrium_reactions_config
-        solver.options["max_iter"] = 20
-        results = solver.solve(model)
+    def test_solve(self, model_solve):
+        _, results = model_solve
         assert results.solver.termination_condition == TerminationCondition.optimal
         assert results.solver.status == SolverStatus.ok
 
     @pytest.mark.component
-    def test_solution_inherent(self, inherent_reactions_config):
-        model = inherent_reactions_config
+    def test_solution(self, model_solve):
+        model, _ = model_solve
 
-        assert pytest.approx(298, rel=1e-5) == value(
+        assert pytest.approx(298, rel=1e-4) == value(
             model.fs.unit.outlet.temperature[0]
```

Comment:
Did this tolerance need to be changed? I noticed you did not update any of the other tolerances.

Reply:
Thanks for spotting that, I think you might be right. Let me try to change it back to `1e-5` and see if it does make a difference.